hoot-core/src/main/cpp/hoot/core/conflate/UnifyingConflator.cpp

```diff
@@ -220,7 +220,7 @@ void UnifyingConflator::apply(shared_ptr<OsmMap>& map)
   //#warning validateConflictSubset is on, this is slow.
   //_validateConflictSubset(map, _matches);
 
-  LOG_DEBUG("Post constraining match count: " << _matches.size());
+  LOG_INFO("Post constraining match count: " << _matches.size());
```

Comment:
I did initially want this at info, but thinking about it more anything with numeric content probably belongs at debug most of the time.  

I think following the convention that info's tip off the beginning of each noteworthy operation and then at the end of the operation the result messages stay as debug is probably the best way to stay consistent throughout.  

I think I made it info initially b/c you jump straight from "conflating" to "exporting" or something like that w/o knowing anything about what happened with the conflation.  I was in a situation where if I could have just seen "0 matches" logged at info level from the start, I probably would have figured out the problem more quickly vs. spending hours of sifting through a ton of debug and trace....but now that I know where some of these important log statements occur, I guess its not as big of a deal.  After two straight days of pulling my hair out for why those datasets weren't conflating, I reworked a ton of logging to make problems a little easier to catch.